source/ffmpeg/tools.cpp

```diff
@@ -280,7 +280,7 @@ std::vector<AVPixelFormat> tools::get_software_formats(const AVPixelFormat* list
 
 	fmts.push_back(AV_PIX_FMT_NONE);
 
-	return std::move(fmts);
+	return fmts;
```

Comment:
**Performance:** `std::move` may not be the default on all compilers and c++ libraries.